src/main/kotlin/kweb/Kweb.kt

```diff
@@ -364,7 +366,11 @@ class Kweb private constructor(
             for (plugin in plugins) {
                 //this code block looks a little funny now, but I still think moving the message creation out of Kweb.callJs() was the right move
                 val js = plugin.executeAfterPageCreation()
-                callJs(Server2ClientMessage(yourId = kwebSessionId, js = js), javascript = js)
+                //A plugin with an empty js string was breaking functionality.
```

Comment:
Was this the cause of #189? Should this throw an exception rather than ignoring it silently?

Reply:
This was not the cause of #189. I also don't think it needs an exception. I think it should be ok for plugin's to have an empty executeAfterPageCreation() field. We just need this check so they don't cause problems.